lib/authn/util/poll.ts

```diff
@@ -179,11 +193,14 @@ export function getPollFn(sdk, res: AuthnTransactionState, ref) {
           }
         })
         .catch(function(err) {
+          const isTooManyRequests = err.xhr &&
+            (err.xhr.status === 0 || err.xhr.status === 429);
+          const isNetworkError = isAuthApiError(err) && err.message === 'Load failed';
+          const isTimeout = isAuthApiError(err) && err.message === 'Load timeout';
```

Comment:
Looks brittle. Check that this works cross-browser (at least doesn't cause regression)